src/modules/fancyzones/lib/SecondaryMouseButtonsHook.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <functional>
+
+class SecondaryMouseButtonsHook
+{
+public:
+    SecondaryMouseButtonsHook(std::function<void()>);
+    void enable();
+    void disable();
+
+private:
+    static HHOOK m_hHook;
```

Comment:
We could confuse this for a member variable because of `m_`, but actually it's a static member. Not sure whether we have a convention for static members.